mmcls/models/losses/cross_entropy_loss.py

```diff
@@ -18,15 +18,63 @@ def cross_entropy(pred, label, weight=None, reduction='mean', avg_factor=None):
     return loss
 
 
+def binary_cross_entropy(pred,
+                         label,
+                         weight=None,
+                         reduction='mean',
+                         avg_factor=None):
+    """Calculate the binary CrossEntropy loss with logits.
+
+    Args:
+        pred (torch.Tensor): The prediction with shape (N, C), where C is the
+            number of classes.
+        label (torch.Tensor): The learning label with shape (N, C), where C is
+            the number of classes.
+        weight (torch.Tensor, optional): Weight of loss. Defaults to None.
+        reduction (str, optional): The method used to reduce the loss.
+            Options are "none", "mean" and "sum". Defaults to 'mean'.
+        avg_factor (int, optional): Average factor that is used to average
+            the loss. Defaults to None.
+
+    Returns:
+        torch.Tensor: The calculated loss
+    """
+    assert pred.dim() == label.dim()
+
+    loss = F.binary_cross_entropy_with_logits(pred, label, reduction='none')
+    loss = loss.sum(dim=1)
+
+    # apply weights and do the reduction
+    if weight is not None:
+        weight = weight.float()
+    loss = weight_reduce_loss(
+        loss, weight=weight, reduction=reduction, avg_factor=avg_factor)
+    return loss
```

Comment:
I thought if `reduction` is `'none'`, shape of loss must be same as input shape, but if it is not `'none'`, it should be averary or summary all elements. 
FYI. https://pytorch.org/docs/stable/generated/torch.nn.BCELoss.html?highlight=bce#torch.nn.BCELoss